lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -339,19 +338,14 @@ def draw(self, renderer):
         self.offsetText.set_ha(align)
         self.offsetText.draw(renderer)
 
-        # Draw grid lines
         if self.axes._draw_grid and len(ticks):
-            # Grid points at end of one plane
-            xyz1 = xyz0.copy()
-            newindex = (index + 1) % 3
-            xyz1[:, newindex] = maxmin[newindex]
-
-            # Grid points at end of the other plane
-            xyz2 = xyz0.copy()
-            newindex = (index + 2) % 3
-            xyz2[:, newindex] = maxmin[newindex]
-
-            lines = np.stack([xyz1, xyz0, xyz2], axis=1)
+            # Grid lines go from the end of one plane to the plane intersection
```

Comment:
Would this be clearer?
```suggestion
            # Grid lines go from the end of one plane through the plane intersection
```

Reply:
yes and yes.  fixed.